tests/ergo_run_test.go

```diff
@@ -556,7 +581,14 @@ func TestConfigDynamicWindows(t *testing.T) {
 	if *getOS() != "windows" {
 		t.Skip("Not running windows run specific tests")
 	}
-	err := setupErgo()
+
+	c, err := newConfigFromFile("./.ergo")
+	if err != nil {
+		t.Fatalf("Unable to copy config file: %v", err)
+	}
+	defer c.clean()
+
+	err = setupErgo(c.filePath)
```

Comment:
These lines are repeated several times. Perhaps a method (like _setupErgoWithTempConfig_) would be beneficial.
This could return a config that you can use later to close run ergo.